models/public/quartznet/quartznet.md

```diff
@@ -0,0 +1,75 @@
+# quartznet
+
+## Use Case and High-Level Description
+
+QuartzNet model is a pair of encoder and decoder. QuartzNet’s design is based on the Jasper architecture,
+which is a convolutional model trained with Connectionist Temporal Classification (CTC) loss.
+The model was trained in NeMo on multiple datasets: LibriSpeech,
+Mozilla Common Voice, WSJ, Fisher, Switchboard, and NSC Singapore English.
+For details see [repository](https://github.com/NVIDIA/NeMo), [paper](https://arxiv.org/pdf/1910.10261.pdf).
+
+## Accuracy
+
+| Metric                       | Value |
+| ---------------------------- | ----- |
+| WER @ Librispeech test-clean | 3.86% |
+
+## Specification
+
+| Metric           | Value              |
+| ---------------- | ------------------ |
+| Type             | Speech recognition |
+| GFLOPs           | 2.4195             |
+| MParams          | 18.8857            |
+| Source framework | PyTorch\*          |
+
+### Input
+
+#### Original model
+
+Audio signal, name - `audio_signal`,  shape - `1, 64, 128`, format is `B, N, C` where:
+
+- `B` - batch size
+- `N` - number of audio frames
+- `C` - clip duration
+
```

Comment:
This is lacking some important information about the expected data: what is the format of the samples? (-1..1? 0..65535?) What is the sample rate?

Reply:
Updated input information